proto/serverless/proto/instrumentation/tags/v1/tags.proto

```diff
@@ -0,0 +1,295 @@
+    string sdk_name = 1;
+    // The version of the Serverless SDK used to instrument.
+    string sdk_version = 2;
+  }
+
+  SdkTags sdk = 5;
+
+  // An optional environment that can be attached.
+  optional string environment = 16;
+  // An optional namespace that can be attached.
+  optional string namespace = 17;
+
+
+}
+
+message AwsTags {
+  // The AWS Account ID that instrumentation was performed in.
+  string account_id = 1;
+  // The ARN of the instrumented AWS resource.
+  string resource_arn = 2;
+
+  // The optional X-Ray TraceId from AWS. This will only be available if X-Ray was enabled in the instrumented Account & Service.
+  optional string xray_trace_id = 16;
+}
+
+message AwsApiGatewayTags {
+  // The AWS Account ID of the API GW.
+  string account_id = 1;
+  // The unique id used for the API GW.
+  string api_id = 2;
+  // The stage of the API GW endpoint that was called.
+  string api_stage = 3;
+
+  message AwsApiGatewayRequestTags {
+    // The unique API GW Request ID.
+    string id = 1;
+    // The request time in milliseconds from epoch.
+    fixed64 time_epoch = 2;
+    // The HTTP protocol of the request.
+    string protocol = 3;
+    // The domain name of the request.
+    string domain = 4;
+    // JSON string containing Request Headers
+    string headers = 5;
+    // The HTTP method of the request.
+    string method = 6;
+    // The HTTP Path of the request.
+    string path = 7;
+
+
+    // JSON string containing Request Path Parameters
+    optional string path_parameters = 16;
+    // JSON string contain Query String Parameters
+    optional string string_parameters = 17;
+  }
+
+  AwsApiGatewayRequestTags request = 15;
+
+}
+
+message AwsLambdaTags {
+  // The architecture of the Lambda function, currently either amd64 or arm64.
+  string arch = 1;
+  // Was the invocation a cold start?
+  bool coldstart = 2;
+  // The event type for the invocation.
+  string event_type = 3;
+  // The event source for the invocation.
+  string event_source = 4;
+  // The Log Group for the Lambda Function.
+  string log_group = 5;
+  // The Log Stream for the invocation.
+  string log_stream_name = 6;
+  // The Max Memory that is configured for the Lambda Function.
+  fixed64 max_memory = 7;
+  // The Lambda Function name.
+  string name = 8;
+  // The Request ID for the invocation.
+  string request_id = 9;
+  // The request time in milliseconds since epoch.
+  fixed64 request_time_epoch = 10;
+  // The Lambda Function version.
+  string version = 11;
+
+  // Was the invocation an error?
+  bool error = 14;
+  // Was the invocation a timeout?
+  bool error_timeout = 15;
+  // Optional error description.
+  optional string error_culprit = 16;
+  // Optional error exception message.
+  optional string error_exception_message = 17;
+  // Optional error exception stacktrace.
+  optional string error_exception_stacktrace = 18;
+  // Optional error exception type.
+  optional string error_exception_type = 19;
```

Comment:
Those properties feel not clear to me.

For outcome cases as `error:initialization`, `error:unhandled` and `error:handled`, we can provide  additionally `error_message`and `error_stack`, and that's basically it.

Such setup I think should work well towards any runtime.

Reply:
These are what existed in the current extension. I'm not tied to them, but not sure if they are needed. @ac360 @Danwakeem can you guys confirm these can be simplified?

Reply:
I think the only thing we don't take advantage of in console UI is `error_culprit`.

- The `error_exception_message` is used to show just an error message without a stack trace.
- The `error_exception_stacktrace` is used to display the full stack trace
- The `error_exception_type` is where we were adding where the error originated from (I think this is what @medikoo is talking about with `error:initalization`, `error:unhandled`, etc.)

I think each one of those fields will be useful and I am not sure if there is much we could remove from this list other than `error_culprit` but I could be misunderstanding what the suggestion is here 🤔

Reply:
Ok so `error_exception_type` can be resolved from the `outcome` enum (assuming we'll stick to such tag name), and having that we're good with just first two